src/test/java/it/pagopa/pn/paperchannel/middleware/db/entities/PnPaperChannelCostTest.java

```diff
@@ -0,0 +1,102 @@
+package it.pagopa.pn.paperchannel.middleware.db.entities;
+
+import it.pagopa.pn.paperchannel.model.Range;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+
+class PnPaperChannelCostTest {
+    private String tenderId;
+    private String product;
+    private String lot;
+    private String zone;
+    private String deliveryDriverName;
+    private String deliveryDriverId;
+    private BigDecimal dematerializationCost;
+    private List<Range> rangedCosts;
+    private Instant createdAt;
+
+
+    @BeforeEach
+    void setUp(){
+        this.initialize();
+    }
+
+    @Test
+    void toStringTest() {
```

Comment:
Eliminiamo tutti i test sul toString. In generale i metodi autogenerati da Lombok non inficiano sulla copertura, grazie al file lombok.config (se invece avevamo scritto il toString a mano, era inficiava sulla copertura). Facciamolo per tutti i test scritti